fluss-flink/fluss-flink-common/src/main/java/com/alibaba/fluss/flink/tiering/source/state/TieringLogSplitState.java

```diff
@@ -0,0 +1,46 @@
+/*
+ *  Copyright (c) 2025 Alibaba Group Holding Ltd.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package com.alibaba.fluss.flink.tiering.source.state;
+
+import com.alibaba.fluss.flink.tiering.source.split.TieringLogSplit;
+
+/** The state for {@link TieringLogSplit}. */
+public class TieringLogSplitState extends TieringSplitState {
+
+    /** The next log offset to sync(tier) to lake, should only be monotonically increasing. */
+    private long nextOffset;
+
+    public TieringLogSplitState(TieringLogSplit tieringSplit, long nextOffset) {
+        super(tieringSplit);
+        this.nextOffset = nextOffset;
+    }
+
+    public void nextOffset(long nextOffset) {
```

Comment:
Do we need to update `nextOffset` since we won't store the state in our tiering service so that can't get the latest emitted record's offset . Seems update `nextOffset` has no meanings.